.github/workflows/test.yml

```diff
@@ -27,14 +27,27 @@ jobs:
         (New-Object System.Net.WebClient).DownloadFile("http://prdownloads.sourceforge.net/swig/swigwin-4.0.1.zip","swigwin-4.0.1.zip");
         Expand-Archive .\swigwin-4.0.1.zip .;
         echo "$((Get-Item .).FullName)/swigwin-4.0.1" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
-    - name: Install APT dependencies
-      if: "startsWith(runner.os, 'ubuntu')"
+    - name: Setup Display
+      if: "startsWith(runner.os, 'Linux')"
       run: |
-        sudo apt-get install -y git build-essential apt-utils wget libfreetype6 libpng-dev libopenblas-dev gcc gfortran libsnappy-dev
+        sudo apt -y install xvfb libgtk-3-dev freeglut3-dev
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip wheel
+        pip install wxPython
+        pip install pyo
+        pip install h5py
         pip install . -r requirements.txt
+        pip install -e .
+    - name: Run eegnb
+      if: "startsWith(runner.os, 'Linux')"
+      run: |
+        Xvfb :0 -screen 0 1024x768x24 -ac +extension GLX +render -noreset &> xvfb.log &
```

Comment:
required to fix: `pyglet.canvas.xlib.NoSuchDisplayException: Cannot connect to "None"`
(thanks to: https://github.com/duckietown/gym-duckietown/issues/123#issuecomment-459638866)